CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 3.0.0
+  * Complete tap refactoring [#64](https://github.com/singer-io/tap-harvest/pull/64)
```

Comment:
IMO discovery support and other major feature updates should get mentioned here.